src/ontology/pato-edit.obo

```diff
@@ -21953,7 +21953,7 @@ created_by: http://orcid.org/0000-0001-7258-9596
 [Term]
 id: PATO:0070012
 name: bitufted cell morphology
-def: "A cell morphology that inheres in neurons which have two clusters of branches that originate directly from the soma and extend in opposite directions and axons that form a plexus which spreads widely. Compared to bipolar morphology, bitufted morphology has branching that occur close to the soma." [doi:10.1016/b978-0-12-369497-3.10004-4, PMID:18568015]
+def: "A cell morphology that inheres in neurons which have two clusters of primary dendrites that originate directly from the soma and extend in opposite directions, and axons that form a plexus which spreads widely. Compared to the narrow vertical axonal projection of bipolar morphology, and the 'horse-tail' axonal cluster of double boquet cells, bitufted morphology has wide horizontal axonal spans, and branching that occur close to the soma. Their vertical projection is also less extensive and crosses mostly to neighboring layers." [doi:10.1016/b978-0-12-369497-3.10004-4, PMID:15378039, PMID:18568015]
```

Comment:
My reading was that they originate on opposite sides of the soma, because you could imagine two primary dendrites that originate on the same pole and then extend in opposite directions.

Reply:
oh yeah, didn't think about that, will fix, thanks for spotting :)